testsuite/classlibrary/TestLcmGcd.sc

```diff
@@ -0,0 +1,156 @@
+
+TestLcmGcd : UnitTest {
+
+
+	// lattice theoretic laws
+	// https://en.wikipedia.org/wiki/Least_common_multiple#Lattice-theoretic
+
+	callTest_commutative_lcm { |a, b|
+		var x = lcm(a, b);
+		var y = lcm(b, a);
+		this.assertEquals(x, y, "lcm(%, %) = lcm(%, %) should be valid"
```

Comment:
i would prefer if this was a single test that checks all 81 test cases and just says "`lcm(x,y) = lcm(y,x)`". that would make the post window output cleaner.